catalog-info.yaml

```diff
@@ -0,0 +1,37 @@
+# This file records information about this repo. Its use is described in OEP-55:
+# https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0055-proc-project-maintainers.html
+
+apiVersion: backstage.io/v1alpha1
+# (Required) Acceptable Values: Component, Resource, System
+# A repo will almost certainly be a Component.
+kind: Component
+metadata:
+  name: 'xblock-drag-and-drop-v2'
+  description: "An XBlock implementing a friendly drag-and-drop style problem"
+  links:
+    - url: "https://github.com/openedx/xblock-drag-and-drop-v2"
```

Comment:
You don't need to link to the source code, that is done automatically by backstage.